frontend/index.html

```diff
@@ -30,7 +30,7 @@
         Sentry.init({
             dsn: "https://2449cc52f1c847c6aa361dbde07e49c8@o4503948031426560.ingest.sentry.io/4503948034703360",
             environment: "local",
-            release: "0621722"
+            release: "6976421"
```

Comment:
I usually don’t commit this one. It will always change and just clutter. Not found a good solution for this not to show up in git status

Reply:
Ah yea I didn't know if it needed to be updated but saw no issue if it was so just committed it, in future I'll make sure not to